pkg/settings/settings.go

```diff
@@ -29,6 +29,9 @@ var (
 	SupportBundleImage           = NewSetting("support-bundle-image", "rancher/support-bundle-kit:v0.0.3")
 	SupportBundleImagePullPolicy = NewSetting("support-bundle-image-pull-policy", "IfNotPresent")
 	DefaultStorageClass          = NewSetting("default-storage-class", "longhorn")
+	CpuOvercommitPercentage      = NewSetting("cpu-overcommit-percentage", "1000")    // KubeVirt's default is "10"x ratio. To align with others, we change it to percentage.
+	MemoryOvercommitPercentage   = NewSetting("memory-overcommit-percentage", "100")  // KubeVirt's default
+	StorageOvercommitPercentage  = NewSetting("storage-overcommit-percentage", "200") // Longhorn's default
```

Comment:
prefer to use one setting but a JSON string config instead. e.g "overcommit-config": {"cpu-overcommit": "200", "memory-overcommit": "200", ....}